migration/migrator/data/submitty_db.sql

```diff
@@ -98,10 +98,10 @@ CREATE FUNCTION public.sync_courses_user() RETURNS trigger
                 IF (TG_OP = 'INSERT') THEN
                     -- FULL data sync on INSERT of a new user record.
                     SELECT * INTO user_row FROM users WHERE user_id=NEW.user_id;
-                    query_string := 'INSERT INTO users (user_id, user_numeric_id, user_givenname, user_preferred_givenname, user_familyname, user_preferred_familyname, user_email, user_email_secondary, user_email_secondary_notify, user_updated, instructor_updated, user_group, registration_section, registration_type, manual_registration) ' ||
```

Comment:
Make sure the `user_email_secondary` and `user_email_secondary_notify` columns are in here for the trigger function to work correctly. Please see PR #8896 which was recently merged.